application.py

```diff
@@ -32,15 +39,16 @@
 
 # Set up database
 
+@app.route("/")
+def index():
+    if 'username' in session:
+        return redirect(url_for('home'))
+    return redirect(url_for("register"))
 
 
 
 @app.route("/register")
-def index():
-
-    if 'username' in session:
-        return redirect(url_for('home'))
-    
+def register():
```

Comment:
I believe I mentioned the route should be /register and not /User in many of my feedbacks, why hasn't there being a change. /register on POST should create user, on GET should render registration page.